shared/params/mainnet_config.go

```diff
@@ -131,6 +131,9 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	WeakSubjectivityPeriod:    54000,
 	PruneSlasherStoragePeriod: 10,
 
+	// Weak subjectivity values.
+	SafetyDecay: 1 / 10,
```

Comment:
0?

Reply:
oh shit, let me fix this. 

note: need a better test